tensorflow/contrib/legacy_seq2seq/python/ops/seq2seq.py

```diff
@@ -792,6 +792,7 @@ def embedding_attention_seq2seq(encoder_inputs,
                                 num_encoder_symbols,
                                 num_decoder_symbols,
                                 embedding_size,
+                                encoder_cell=None,
```

Comment:
Hi, I have added the description of new arg 'encoder_cell' in comment. Please have a check.